satpy/readers/ahi_hsd.py

```diff
@@ -587,6 +626,10 @@ def convert_to_radiance(self, data):
         else:
             gain = self._header["block5"]["gain_count2rad_conversion"][0]
```

Comment:
In some places we use `gain`, in others `slope`. Please choose one and apply it to the entire reader.